docs/smartsheet.md

```diff
@@ -0,0 +1,23 @@
+# Integrating the Smartsheet Provider
+
+## Example
+
+```csharp
+services.AddAuthentication(options => /* Auth configuration */)
+        .AddSmartsheet(options =>
+        {
+            options.ClientId = "my-client-id";
+            options.ClientSecret = "my-client-secret";
+
+            // At least one scope must be passed
+            options.Scope.Add("READ_SHEETS");
```

Comment:
Is there such a thing as a "default" scope that everyone would always be expected to provide? If so, then it would be best to configure it as a sensible default in the options class, and then we don't need to add this additional documentation.

If there is no sensible default, then this is fine as-is.

Reply:
There is no default scope specified and from the [list](https://smartsheet.redoc.ly/#section/Authentication/OAuth2) I don't see a sensible default either.